src/in/shick/diode/threads/DownloadThreadsTask.java

```diff
@@ -96,10 +113,12 @@ public DownloadThreadsTask(Context context, HttpClient client, ObjectMapper om,
         mOm = om;
         mSortByUrl = sortByUrl;
         mSortByUrlExtra = sortByUrlExtra;
-        if (subreddit != null)
+        mDTTSavedURI = null;
+        if (subreddit != null) {
             mSubreddit = subreddit;
-        else
+        } else {
```

Comment:
Great comments where helpful *and* cleaning up sloppy implicit braces, woohoo!

Reply:
Sorry, I get anal about formatting.

Reply:
Nothing to be sorry about, they're good changes.